microsoft-edge/webview2/index.md

```diff
@@ -84,16 +84,21 @@ WebView2 apps can run on the following versions of Windows:
 *  Windows 10 IoT Enterprise LTSC x32 2019
 *  Windows 10 IoT Enterprise LTSC x64 2019
 *  Windows 10 IoT Enterprise 21h1 x64
-*  Windows 8.1
-*  Windows 7 \*\*
 *  Windows Server 2022
 *  Windows Server 2019
 *  Windows Server 2016
-*  Windows Server 2012
+
+WebView2 SDK version 1.0.1519.0 and higher doesn't support the following versions of Windows:
+
+*  Windows 8/8.1
+*  Windows 7
 *  Windows Server 2012 R2
-*  Windows Server 2008 R2 \*\*
+*  Windows Server 2012
```

Comment:
@nishitha-burman - it wasn't explicitly clear in blog to remove these OSs for wv2

Reply:
@champnic - do these 3 Server items belong here, as not supported, or in the Supported list above this section?  The paragraph at blog about Server OSs mentions Edge, not wv2.

https://blogs.windows.com/msedgedev/2022/12/09/microsoft-edge-and-webview2-ending-support-for-windows-7-and-windows-8-8-1/ - 
Microsoft **Edge** version 109 will also be the last supported version for Windows **Server** 2008 R2, Windows **Server** 2012, and Windows **Server** 2012 R2. Internet Explorer 11 will remain supported on those operating systems for as long as they are in support.

Reply:
Sorry yes - these are not supported and should be in the not supported list. (My "remove" in my comment was to "remove from the supported list", sorry for the confusion)